main.py

```diff
@@ -20,9 +20,17 @@
     "https://ssr.gophie.cam",
 ]
 
+ENGINES = {
+    "netnaija": "NetNaija",
```

Comment:
@deven96, we will need to add all the engines actual Database names here in lowercase

Reply:
@MeNsaaH I've replaced it using sqlalchemy func.lower() to lower the database strings

Reply:
The problem is existing movies are stored using the old implementation. We will have to do a big database rewrite to update the data